packages/ckeditor5-dev-env/lib/release-tools/utils/cli.js

```diff
@@ -34,7 +34,7 @@ const cli = {
 		};
 
 		return inquirer.prompt( [ confirmQuestion ] )
-			.then( answers => answers.confirm );
+			.then( answers => answers.confirm ? Promise.resolve() : Promise.reject() );
```

Comment:
This is like throwing an error. Error means that something didn't work. So this change is incorrect.